### DISCUSSION TITLE
Mail app can't connect to imap server

### DISCUSSION THREAD

**Discussion by eustachy84** [nextcloud/all-in-one#2647] [Questions] [answered]
Hi

I try to configure mail app on AIO. It cant connect to my email server over domain name (served from another host), or internal ip. Ive tried to coonect to imap gmail (setting application pass first) and it cant connect. Run Firewall log on router, and there isnt any try to connect from this host to imap server (internal or external). Can You help?
Notification from internal(not mail app) (sended from privete mail host) works as charm...

**Comment by szaimen** [comment, marked as answer]
Hi, this sounds like something that you would need to fix in your networking infrastucture...

Also please ask this question at https://help.nextcloud.com
